roles/intel/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
     name: "{{ item }}"
     state: present
   with_items:
-    - drm-61-kmod
+    - drm-66-kmod
     - gpu-firmware-kmod
```

Comment:
while here, I think we can use `fwget` to install the device specified firmware instead of the gpu-firmware-kmod meta package contains everything.